### DISCUSSION TITLE
32-Bit Windows Machine Not able to ship eventlog to SecurityOnion-v2-3

### DISCUSSION THREAD

**Discussion by Vic-in** [Security-Onion-Solutions/securityonion#7285] [Unsupported Versions] [answered]
Hi, 
I have a problem that hope the community is able to provide some lead.
I have installed Security Onion version 2-3 on CentOS 7 as "Standalone" mode. I configured to ship log using logstash. 
'
Working
*********
Installed with Winlogbeat version 8.0, I am able to see Sysmon log shipped Windows client running either Windows 7 or 10 if the machine are 64-bit architecture. 

Not Working
***************
Winlogbeat 8.0 does not come with 32-bit version, so I have tried Winlogbeat version 6.8, 7.12 and 7.17 for 32-bit machine. The installation process and test config ok. However, I am not able to see Sysmon log from these client. Network connection to Security Onion should be good as I can see other DNS log from these 32-bit client from Kinaba GUI.

Thanks in advance for any help given to this thread.

**Comment by dougburks** [comment, marked as answer]
Make sure that you're using the version of Winlogbeat that matches the Elastic version in your Security Onion deployment. Then verify that sysmon is configured correctly and logging properly to the Windows box itself. If Winlogbeat 8 doesn't support 32-bit, then it would seem that they are dropping support for 32-bit and you will want to consider upgrading that Windows box to 64-bit or looking at a different endpoint agent.

**Reply by Vic-in** [reply]
Hi dougburks,

Look like the solution is to run on 64-bit machine.
Thank you for you reply.
